### PR TITLE
fix setting direction of remote units via zeus

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- title

It has been years. They will never fix this. This work around is very simple.

### CODE REVIEW

**Comment by PabstMirror** [comment]
maybe add 
`if (missionNamespace getVariable [QGVAR(disableZeusDirFix), false]) exitWith {};`
Just in case this causes problems or is undesired?

**Comment by commy2** [comment]
We could store the ehid in a object namespace variable on the logic.

File: addons/common/XEH_preInit.sqf [hunk 1]
```diff
@@ -37,4 +37,22 @@ GVAR(addons) = _addons;
...
+// fix changing direction of remote units not working with zeus
+["ModuleCurator_F", "init", {
+    params ["_logic"];
+
+    _logic addEventHandler ["CuratorObjectEdited", {
+        params ["", "_unit"];
+
+        if (!local _unit) then {
+            private _dir = getDir _unit;
+            [_unit, _dir] remoteExec ["setDir", _unit];
```
Comment (shukari) [inline addons/common/XEH_preInit.sqf]: With doWatch you can also let tanks turn the gun. Would be nice to have ;D

Comment (Cuel) [reply addons/common/XEH_preInit.sqf]: The gun turns when you rotate them anyway?

Comment (commy2) [reply addons/common/XEH_preInit.sqf]: The turret is rotating fine with this fix without having to do doWatch.

**Pull request merged**